repos/rust-lang/rust-forge.toml

```diff
@@ -0,0 +1,19 @@
+org = 'rust-lang'
+name = 'rust-forge'
+description = 'Information useful to people contributing to Rust'
+bots = ["rustbot"]
+
+[access.teams]
+community = "maintain"
+compiler = "maintain"
```

Comment:
Does that include subteams? If not, maybe adding `compiler-contributors` could be helpful, as there are twice as many people?

Reply:
It does not include subteams. I went ahead and added compiler-contributors.